test/config/SettingTest.java

```diff
@@ -1,8 +1,12 @@
 package config;
 
-import org.junit.Assert;
-import org.junit.jupiter.api.Test;
```

Comment:
Please remove the jupiter library from the `JARS` folder, it is important to clean up the project to use only the required libraries.